src/components/ApplicationListView/ApplicationListView.tsx

```diff
@@ -59,6 +65,25 @@ const ApplicationListView: React.FC = () => {
         </GettingStartedCard>
       )}
       <PageLayout
+        breadcrumbs={[
+          <Badge key="badge" isRead>
+            WS
+          </Badge>,
+          <span key="badge-divider" className={styles.breadcrumbItemDivider} />,
+          <BreadcrumbItem key="workspace-link" to="#">
+            <Link className="pf-c-breadcrumb__link" to="#">
+              {workspace}
+            </Link>
+          </BreadcrumbItem>,
+          <WorkspaceSwitcher key="workspace" />,
+          <span key="workspace-divider" className={styles.breadcrumbItemDivider}>
+            ⧸
+          </span>,
+          {
+            path: '',
+            name: 'Applications',
+          },
```

Comment:
I think we need either the `/` divider or `>` divider here, but not both.
![0](https://user-images.githubusercontent.com/20013884/218054404-cd94ed82-5505-4105-b5c0-58f1393476e8.png)
You could replace the breadcrumb object with a BreadcrumbItem component to remove the `>`.

Reply:
Updated it.

![image](https://user-images.githubusercontent.com/9964343/218088544-687c1e4b-3b1c-48c1-86ea-1c7b6bbf8037.png)